### ISSUE TITLE
Language type icon overriding file autocompletion

### ISSUE THREAD

**Issue by tommasoclini** [microsoft/vscode#217387]
Type: <b>Bug</b>

edit a file in C and increase the font size, then try including a header file and activate the autocompletion.

VS Code version: Code 1.90.2 (5437499feb04f7a586f677b155b039bc2b3669eb, 2024-06-18T22:33:48.698Z)
OS version: Linux x64 6.1.0-21-amd64
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-1035G1 CPU @ 1.00GHz (8 x 982)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|1, 1, 1|
|Memory (System)|15.41GB (10.96GB free)|
|Process Argv|--crash-reporter-id 6f97c42c-909d-47ee-a60c-bafb01b1258c|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|gnome|
|XDG_CURRENT_DESKTOP|GNOME|
|XDG_SESSION_DESKTOP|gnome|
|XDG_SESSION_TYPE|wayland|
</details><details><summary>Extensions (16)</summary>

Extension|Author (truncated)|Version
---|---|---
catppuccin-vsc|Cat|3.14.0
arm|dan|1.7.4
esp-idf-extension|esp|1.7.1
cortex-debug|mar|1.12.1
debug-tracker-vscode|mcu|0.0.15
memory-view|mcu|0.0.25
peripheral-viewer|mcu|1.4.6
rtos-views|mcu|0.0.7
cmake-tools|ms-|1.18.42
cpptools|ms-|1.20.5
cpptools-extension-pack|ms-|1.3.0
hexeditor|ms-|1.10.0
stm32-vscode-extension|STM|2.0.1
gnu-mapfiles|tro|1.1.0
cmake|twx|0.0.17
linkerscript|Zix|1.0.4

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscoreces:30445986
vscod805cf:30301675
binariesv615:30325510
vsaa593:30376534
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
a9j8j154:30646983
962ge761:30959799
pythongtdpath:30769146
welcomedialog:30910333
pythonnoceb:30805159
asynctok:30898717
pythonregdiag2:30936856
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
accentitlementst:30995554
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
jchc7451:31067544
chatpanelc:31048052
dsvsc021:30996838
01bff139:31013167
pythoncenvpt:31062603
a69g1124:31058053
dvdeprecation:31068756
dwnewjupyter:31046869
legacy_priority:31077528

```

</details>

**Comment by jrieken** [comment]
repros with `"editor.fontSize": 40`

**Comment by tommasoclini** [comment]
Hi, what do you mean?

**Comment by tommasoclini** [comment]
Only now I understand what you meant by "repros", I tried putting font size to 40 and I had the same issue, do you need a screenshot?

**Comment by jrieken** [comment]
No, I already have a screen shot and just added/commented-with the confg that's needed to reproduce said capture

**Comment by tommasoclini** [comment]
How is it still unfixed in the latest vscode release version?

**Comment by tommasoclini** [comment]
Is this ever going to get fixed?
